Flashbots tracing improvement, interval period lowered

sunce86 opened the pull request:
Improved error handling of the tx submit and cancel functions.

Interval period for simulation lowered from 10 to 5 sec.

Gas price comparison made more strict.

Test plan:
- [x] Make a manual tx on mainnet and make sure tx is mined (`https://etherscan.io/tx/0xb2556a93a97f8c785720a95fd5229f6ac7b0a9edc96d533d9e136671a7846556`)

sunce86 committed:
improved tracing, flashbots speed up

sunce86 committed:
small rename

bh2smith reviewed (approved):
Changes in the API file seem unrelated to the description. Is there any reason you didn't share the test plan transaction here?

sunce86 commented:
> Changes in the API file seem unrelated to the description. Is there any reason you didn't share the test plan transaction here?

Hey Ben, I've added the transaction in the test plan.
Changes in the API file are related to the tracing improvement (error handling), meaning I've added multiple error messages that are traced on the caller side of these functions.
